HiP-FeatureToggle/Models/Entity/FeatureToFeatureGroupMapping.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Entity
+{
+    /// <remarks>
+    /// Since EF Core does not (yet) support many-to-many relationships, the relationship between features
+    /// and feature groups is modelled as two 1-to-many relationships. In other words, this class acts as
+    /// the join table.
+    ///
+    /// [Feature]----------------[FeatureToFeatureGroupMapping]-------------[FeatureGroup]
+    ///          1              n                              m           1
+    /// </remarks>
+    public class FeatureToFeatureGroupMapping
```

Comment:
1. Moving entity-specific model configuration out of DbContext's OnModelCreating method is a good idea which I will respect in the future (a static method would probably suffice, instead of a separate *Map-class but that doesn't really matter)

2. Most entity relations are set-up by convention in EF, for example lines 28/29 are most likely not needed here: https://github.com/HiP-App/HiP-CmsWebApi/blob/ec05dd037c1d59423af97ec3f77c2f66bb406f3e/src/Api/Models/Entity/TopicUser.cs#L28

3. AFAIK, many-to-many relations must use a separate "class in the middle". So this class is conceptually the same as e.g. [TopicUser](https://github.com/HiP-App/HiP-CmsWebApi/blob/ec05dd037c1d59423af97ec3f77c2f66bb406f3e/src/Api/Models/Entity/TopicUser.cs) which creates a many-to-many relation between topics and users